onebusaway-android/src/main/java/org/onebusaway/android/provider/ObaContract.java

```diff
@@ -646,6 +635,33 @@ public static boolean markAsUnused(Context context, Uri uri) {
             values.putNull(ObaContract.Stops.ACCESS_TIME);
             return cr.update(uri, values, null, null) > 0;
         }
+
+        public static int getArrivalFilter(Context context, Uri uri) {
+            ContentResolver cr = context.getContentResolver();
+            Cursor c = cr.query(uri, new String[]{ARRIVAL_FILTER}, null, null, null);
+
+            int result = 0;
+            if (c != null && c.moveToNext()) {
+                result = c.getInt(0);
+            }
+
+            if (c != null) {
+                c.close();
+            }
+
+            return result;
+        }
+
+        public static void setArrivalFilter(Context context, Uri uri, int filterOption) {
```

Comment:
Same here - what you do you think about changing the int parameter to the enum `ArrivalFilter`?